docs/decisions/0003-searchability-and-catalogs.md

```diff
@@ -0,0 +1,42 @@
+# Data should be readily searchable
+
+## Summary
+
+It's not just a good idea, it's technically the law. While CDA currently expose a fair amount of information to search
+it's never entirely clear.
+
+## Opinions
+
+### Opinion 1
+
+Summary: Each data time should support it's own /catalog end point.
+
+@MikeNeilson
+
+The original CDA has the say, `/timeseries` end point provide a catalog if no data is set. I created a /catalog end point
+to attempt to consildate search query parameters. For TimeSeries and Locations this works reasonably well since there
+is parity between the concepts.
+
+However, if we tried to add ratings into the mix, the list of query parameters grows, and it would rather difficult to 
+document which is for what or what changes for each.
+
+To make 'catalog' operations clear, we should create /catalog for each data type that provide for discoverability of that data.
+
+### Opinion 2
+
+Summary: Each datatype under "catalog" should be a full path"
+
+@MikeNeilson
+
+If it makes sense to group all catalogs under catalog, perhaps for grouping in the SWAGGER-UI, making each catalog it's own
+path under `/catalog` instead of the current path parameter is a better approach.
+
+We would maintain the grouping, but each catalog can have it's appropriate search criteria.
```

Comment:
This does seem in opposition to the accept header reasoning. We already implement getAll for most (all?) data types. However, most of them return the full data object rather than a listing of refs/ids. The catalog endpoint is supposed to be the lightweight alternative to a getAll, which seems to be a change in the shape of the data. A `application/json;catalog=true` or something similar.

Reply:
Fair point, I hadn't thought of just using a different content-type of content-type parameter in that situation.

Reply:
In my opinion we haven't done a great job of explaining what a catalog is vs just a bulk-retrieve end-point.  I think they are (or can be) different things.   In some places we've added flags to enable the controllers to returned reduced data fields or added new *Identifier endpoints but that hasn't been done consistently